### PR TITLE
PVP 285 Added dashboards screen

### PR DESCRIPTION
**Pull request by vytjar**
- Created dashboards screen featuring statistics of user steps, calories, BPM, short summary of tasks and goals
- Made statistics screen part of dashboard screen

### CODE REVIEW

File: app/src/main/java/com/pvp/app/ui/screen/statistics/DashboardScreen.kt [hunk 1]
```diff
@@ -0,0 +1,454 @@
...
+}
+
+@Composable
+private fun ActivityRowItem(
+    icon: ImageVector,
+    title: String,
+    value: String,
+) {
+    Box(
+        modifier = Modifier.size(110.dp)
```
Comment (klaidoshka) [inline app/src/main/java/com/pvp/app/ui/screen/statistics/DashboardScreen.kt:126]: Could be single line

File: app/src/main/java/com/pvp/app/ui/screen/statistics/DashboardScreen.kt [hunk 2]
```diff
@@ -0,0 +1,454 @@
...
+                InformationElement(stringResource(R.string.dashboard_healthconnect_not_enabled))
+
+                Spacer(modifier = Modifier.height(16.dp))
+            }
+
+            if (!state.isNotificationEnabled) {
+                InformationElement(stringResource(R.string.dashboard_notification_not_enabled))
+
+                Spacer(modifier = Modifier.height(16.dp))
+
```
Comment (klaidoshka) [inline app/src/main/java/com/pvp/app/ui/screen/statistics/DashboardScreen.kt:204]: Redundant line

File: app/src/main/java/com/pvp/app/ui/screen/statistics/DashboardViewModel.kt [hunk 3]
```diff
@@ -0,0 +1,184 @@
...
+                    user = user
+                )
+            }
+                .collectLatest { state -> _state.update { state } }
+        }
+    }
+
+    suspend fun getCalories(): Double {
+        val today = LocalDate.now()
+
```
Comment (klaidoshka) [inline app/src/main/java/com/pvp/app/ui/screen/statistics/DashboardViewModel.kt:95]: Redundant line

File: app/src/main/java/com/pvp/app/ui/screen/statistics/DashboardViewModel.kt [hunk 4]
```diff
@@ -0,0 +1,184 @@
...
+        return healthConnectService.aggregateSteps(
+            start,
+            end
+        )
+    }
+
+    suspend fun getHeartRate(): Long {
+        val today = LocalDate.now()
+
```
Comment (klaidoshka) [inline app/src/main/java/com/pvp/app/ui/screen/statistics/DashboardViewModel.kt:128]: Redundant line

**Review by klaidoshka — APPROVED** [review]

**Review by AurisTFG — APPROVED** [review]

**Pull request merged**